client/src/constants.ts

```diff
@@ -28,3 +32,16 @@ export const UPDATE_FREQUENCY_LABELS: { [K in UpdateFrequency]: string } = {
   monthly: "Mensuelle (ou plusieures fois pas mois)",
   yearly: "Annuel (ou plusieures fois par an)",
 };
+
+export const GEOGRAPHICAL_COVERAGE_LABELS: {
+  [K in GeographicalCoverage]: string;
+} = {
+  municipality: "Communale",
+  EPCI: "EPCI",
+  departement: "Departementale",
```

Comment:
En anglais ça s'écrit "department", je vais utiliser ça côté back pour être cohérent avec les autres clés

+ manque accent

```suggestion
  department: "Départementale",
```